### ISSUE TITLE
TypeError: Cannot read property 'ModuleKind' of null

### ISSUE THREAD

**Issue by nkbt** [depcheck/depcheck#280]
### Bug Description
```
$ depcheck
/Users/nkbt/mh/feed-router/node_modules/depcheck/dist/parser/typescript.js:15
  module: typescript.ModuleKind.CommonJS,
                     ^

TypeError: Cannot read property 'ModuleKind' of null
    at Object.<anonymous> (/Users/nkbt/mh/feed-router/node_modules/depcheck/dist/parser/typescript.js:15:22)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at /Users/nkbt/mh/feed-router/node_modules/depcheck/dist/constants.js:24:19
    at arrayMap (/Users/nkbt/mh/feed-router/node_modules/lodash/lodash.js:639:23)
error Command failed with exit code 1.
```

Typescript is not listed as a dependency.
We do not use typescript either, so it does not exist.

in depcheck/dist/parser/typescript.js:12 `typescript is included
```
var typescript = (0, _utils.tryRequire)('typescript');
```

### Versions
 - `node -v`: v10.7.0
 - `npm -v`: 6.1.0
 - `depcheck -v`: 0.6.10-beta.1

### Extra info
I guess this

**Comment by nkbt** [comment]
On a side note I suggest publishing beta version as `depcheck@next` or another dist-tag.
See https://docs.npmjs.com/cli/publish `--tag` option

**Comment by mnkhouri** [comment]
@nkbt Thank you for the issue and the suggestion. I've reset `latest` to 0.6.9 and published `0.6.10-beta.1` as `next`, and I'll ensure future beta releases are published under `next` and not `latest`
